.github/workflows/release.yml

```
@@ -3,11 +3,48 @@ name: Release
 on:
   push:
     tags: ["v*"]
+  pull_request:
 
 jobs:
-  github_release:
+  log-updates:
+    name: Log packages to publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the new tag
+        uses: actions/checkout@v1.0.0
```

```suggestion
        uses: actions/checkout@v2
```
The checkout actions v2 by default do a shallow checkout.